Documentation/DocuBlocks/Rest/Indexes/post_api_index_inverted.md

```diff
@@ -176,15 +181,17 @@ Defines how to compress the primary sort data. Possible values:
 The name of an Analyzer to use by default. This Analyzer is applied to the
 values of the indexed fields for which you don't define Analyzers explicitly.
 
+Default: `identity`
+
 @RESTBODYPARAM{features,array,optional,string}
-A list of Analyzer features to use by default. They define what features are
-enabled for the default `analyzer`. Possible features:
+A list of Analyzer features. You can set this option to overwrite what features
+are enabled for the default `analyzer`. Possible features:
```

Comment:
The features have compatibility rules, like position requires frequency. Should we note this?

Reply:
I suppose you refer to this list of dependent features? https://www.arangodb.com/docs/devel/analyzers.html#analyzer-features
I think it's sufficiently covered on that page. I don't want to add a link because we would need to use an absolute URL because of Swagger UI 😞

Reply:
I see. Ok, then.